common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java

```diff
@@ -126,12 +130,27 @@ private void renderDialogue(PoseStack poseStack, String currentDialogue) {
 
             // Render "To continue" message
             // TODO: I'd like to have this better looking, perhaps a clickable button?
+            Optional<Long> millisecondsUntilProgress =
+                    NpcDialogAutoProgressFeature.INSTANCE.millisecondsUntilProgress();
+
+            List<TextRenderTask> renderTaskList;
+
+            renderTaskList = millisecondsUntilProgress
+                    .map(timeUntilProgress -> List.of(
+                            new TextRenderTask("§cPress SNEAK to continue", renderSetting),
+                            new TextRenderTask(
+                                    ChatFormatting.GREEN + "Auto-progress: "
```

Comment:
What about having a bar that's filling up instead? If you start with 2 seconds and just display integer numbers of remaining seconds, it gives very little feedback.

Reply:
Sure